.github/workflows/style.yaml

```diff
@@ -31,14 +31,14 @@ jobs:
 
       - name: Install yara-x
         run: |
-          wget https://github.com/VirusTotal/yara-x/releases/download/v0.10.0/yara-x-v0.10.0-x86_64-unknown-linux-gnu.gzip -O yara-x.gzip
```

Comment:
This is a large version bump v0.10.0 was released in Nov 26, 2024 a v0.15.0 was released yesterday.

Is it necessary to include with the rest of the PR? Not a block, just curious.

Reply:
re-read the description!

Reply:
Ah, yea, this should have been handled with each new version update but it's not tied to the Makefile so it drifted. We should probably fix this in a future PR so that we don't have to manage this separately.

Reply:
Cheers, made an issue for that.